### PR TITLE
Added TIFF checking

### PR DESCRIPTION
**Pull request by MingboPeng**

### CODE REVIEW

**Comment by chriswmackey** [comment]
@MingboPeng ,
I don't see any changes to a .py file but I trust that this is good and I'll merge it.  Are you just changing type hints on the userobject?

**Comment by MingboPeng** [comment]
Hi @chriswmackey , I did change in C#, this is just the gha file:>

**Comment by chriswmackey** [comment]
Ahh, right. I forgot it was written in C#.  I just realized that we should also have you commit a C# file to the src folder just as @mostaphaRoudsari did with the Fly component:
https://github.com/mostaphaRoudsari/ladybug/blob/master/src/Ladybug_Fly.cs

Alternatively, we could just include a link to [your repository](https://github.com/MingboPeng/Ironbug) in the component description.  It would just be good to let people know where to find the source code somewhere on the component.  Let me know if you have a preference of one method over another or any other ideas/suggestions.

On a related note, your newest Colibri component has blown Fly out of the water!  It's really become the best iterator component in the GH community.  Given that I see people moving to use Colibri over Fly and I am personally planning on using Colibri in all of my future tutorial videos, I think I might move Fly to below the Extra Tab for this release.  This will also make things more balanced in the Extra tab with your new awesome component in the same location.

In summary, great job!  Your components are re-shaping AEC workflows!

**Comment by MingboPeng** [comment]
@chriswmackey I am sorry for poor writing of the component descriptions. I will add the source link.
Thanks, and we all are trying to it better.

**Comment by chriswmackey** [comment]
@MingboPeng ,
It's definitely not poor writing and the way that we reference the source code for your component is new and unique given that you have re-written the entire component interface.  So no worries and just send a pull request whenever you're ready today.
